src/busesApp/router.js

```diff
@@ -0,0 +1,52 @@
+import { Router } from 'express';
+import verifyToken from '../middlewares/authJwt.js';
+import CheckPermissionAny from '../middlewares/CheckPermission.js';
+import asyncHandler from '../utils/asyncHandler.js';
+import validate from '../utils/validateMiddleware.js';
+import {
+	postBusHandler,
+	getBusHandler,
+	getSingleBusHandler,
+	putBusHandler,
+	deleteBusHandler,
+} from './controllers.js';
+import updateValidation from './updateValidation.js';
+
+const resource = 'buses';
+
+const busesRouter = Router();
+
+busesRouter.post(
+	'/',
+	verifyToken,
+	CheckPermissionAny(resource),
+	asyncHandler(postBusHandler)
+);
```

Comment:
I think you forgot to validate for the post methods which makes the creation of buses with invalid information acting strange